### PR TITLE
feat(examples): Add `flask-redis` from `awesome-compose` repository

### PR DESCRIPTION
**Pull request by razvand**
Add and update [`flask-redis` app from the `awesome-compose` repository](https://github.com/docker/awesome-compose/tree/master/flask-redis).

There are two commits in the pull request:
1. The first commit copies the contents of the `flask-redis/` directory from the `awesome-compose` repository.
1. The second commit adds and updates contents to make it ready for Unikaft Cloud.

* [x] Update `README.md` for Unikraft Cloud

### CODE REVIEW

File: flask-redis/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:3.12-bookworm AS build
+
+WORKDIR /code
+
+COPY requirements.txt /code
+RUN pip3 install -r requirements.txt --no-cache-dir
+
+COPY . /code
```
Comment (nderjung) [inline flask-redis/Dockerfile:8]: Let's use `/app` instead of `/code`.

File: flask-redis/compose.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+services:
+
```
Comment (nderjung) [inline flask-redis/compose.yaml:2]: Remove blank line between `services` and `redis`.

**Review by nderjung — APPROVED** [review]
Just a couple of suggestions. Otherwise this works as expected. Thanks!

File: flask-redis/compose.yaml (hunk 2 above)
Comment (razvand) [reply flask-redis/compose.yaml]: Did that. Should I also remove the blank line before `web`?

**Review by nderjung — APPROVED** [review]
All good! Thanks!

File: flask-redis/compose.yaml (hunk 2 above)
Comment (nderjung) [reply flask-redis/compose.yaml]: No all good

**Pull request merged**